controllers/apply/form-router/lib/salesforce.js

```diff
@@ -1,78 +1,69 @@
 'use strict';
-const request = require('request-promise-native');
+const got = require('got');
+const FormData = require('form-data');
 const { SALESFORCE_AUTH } = require('../../../../common/secrets');
 
 class Salesforce {
     constructor(apiUrl, accessToken) {
-        this.apiUrl = apiUrl;
         this.apiVersion = 'v45.0';
-        this.headers = { Authorization: `Bearer ${accessToken}` };
-    }
-    get(urlPath) {
-        return request.get({
-            uri: `${this.apiUrl}${urlPath}`,
-            headers: this.headers,
-            json: true,
-        });
-    }
-    postJson(urlPath, body) {
-        return request.post({
-            url: `${this.apiUrl}${urlPath}`,
-            headers: this.headers,
-            json: true,
-            body: body,
+
+        this.query = got.extend({
+            prefixUrl: apiUrl,
+            headers: {
+                'authorization': `Bearer ${accessToken}`,
+                'user-agent': 'tnlcf-salesforce',
+            },
         });
     }
+
     submitFormData({ application, meta }) {
-        return this.postJson('/services/apexrest/FormData/', {
-            meta: meta,
-            application: application,
-        });
+        return this.query
+            .post('services/apexrest/FormData/', {
+                body: { meta, application },
+            })
+            .json();
     }
-    async contentVersion({ recordId, attachmentName, versionData }) {
-        return request.post({
-            url: `${this.apiUrl}/services/data/${this.apiVersion}/sobjects/ContentVersion`,
-            headers: this.headers,
-            formData: {
-                entity_content: {
-                    value: JSON.stringify({
-                        FirstPublishLocationId: recordId,
-                        ReasonForChange: `Application attachment ${attachmentName}`,
-                        PathOnClient: attachmentName,
-                    }),
-                    options: {
-                        contentType: 'application/json',
-                    },
-                },
-                VersionData: versionData,
-            },
-        });
+
+    contentVersion({ recordId, attachmentName, versionData }) {
```

Comment:
Removed the `async` from this as we don't use `await` inside the body of this method, so it isn't needed.